src/components/CustomModal/index.tsx

```diff
@@ -64,7 +64,7 @@ const CustomModal: React.FC<CustomModalProps> = ({
             onCancel={closeHandler}
             closable={false}
             title={title}
-            footer={footer}
+            footer={footerButtons ? footer : null}
```

Comment:
preview modal has no footer

Reply:
I think it makes more sense to do this where you define the footer. and this line would remain unchanged